Purchases/SubscriberAttributes/RCPurchases+SubscriberAttributes.h

```diff
@@ -14,13 +14,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface RCPurchases (SubscriberAttributes)
 
-- (void)_setAttributes:(NSDictionary<NSString *, NSString *> *)attributes;
-- (void)_setEmail:(nullable NSString *)email;
-- (void)_setPhoneNumber:(nullable NSString *)phoneNumber;
-- (void)_setDisplayName:(nullable NSString *)displayName;
-- (void)_setPushToken:(nullable NSData *)pushToken;
-- (void)_setPushTokenString:(nullable NSString *)pushToken;
-
 - (void)configureSubscriberAttributesManager;
 - (RCSubscriberAttributeDict)unsyncedAttributesByKey;
 - (void)markAttributesAsSyncedIfNeeded:(nullable RCSubscriberAttributeDict)syncedAttributes
```

Comment:
Do you think we should move these too? They would be private in RCPurchases @aboedo

Reply:
I wouldn't unless there's a need - do we get warnings for them?

Reply:
No

Reply:
then I'd say keep them here, the main implementation is pretty overloaded